.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "lib/llvm/src"]
 	path = lib/llvm/src
 	url = https://github.com/llvm/llvm-project
-	branch = release/3.9.x
```

Comment:
Does this default to head now because of this?

Reply:
The `branch = release/3.9.x` causes the submodule to track the branch, just deleting this would cause if to track the master branch, IIRC. So not only is `branch = release/3.9.x` deleted here in .gitmodules, but the lib/llvm/src commit now points at a specific commit, d0d8eb2e5415b..., which is what is checked out when doing a `git submodule update --init`:
```
$ git -C lib/llvm/src log -1 --pretty=oneline
d0d8eb2e5415b8be29343e3c17a18e49e67b5551 (HEAD, tag: llvmorg-7.0.1-rc3, tag: llvmorg-7.0.1) Merging r345839:
```

Reply:
Ah, I missed that change. Thanks.